README.md

```diff
@@ -153,6 +166,34 @@ To run the test scenario:
 cargo run --release --package zcash_tx_tool --bin zcash_tx_tool test-orchard-zsa
 ```
 
+### Orchard-ZSA Three Party Scenario
+
+This test scenario ([src/commands/test_three_party.rs](src/commands/test_three_party.rs)) is a three-party setting which performs the following steps:
+
+1. **Issue an Asset**: Create and issue a new ZSA.
+2. **Transfer the Asset (Twice)**: Send the issued ZSA to another account, and then from that account to a third account.
+3. **Burn the Asset**: The third account burns the ZSA.
+
+To run the test scenario:
+
+```bash
+cargo run --release --package zcash_tx_tool --bin zcash_tx_tool test-three-party
+```
+
+### Creating your own scenario
+It is also possible to construct your own scenario in a manner similar to these. 
+To do so, copy one of the test scenario files to a new file in the same location and make the changes to fit your setting.
+
+To allow this new file to be run, make the following changes to [commands.rs](src/commands.rs):
+* Add the module corresponding to your new file to the start of [commands.rs](src/commands.rs).
+* Add an analogous new variant to the `AppCmd` enum.
+
+You should then be able to run your scenario via (assuming `test-scenario` is the name of your scenario):
+```bash
+cargo run --release --package zcash_tx_tool --bin zcash_tx_tool test-scenario
+```
+
+
```

Comment:
looks good, just make sure the headers and subheaders levels are consisten across the entire file